package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "find test -name '*.js' | xargs node"
```

Comment:
Running `tap` here did not seem to work as there was an ESM error. Was also very slow compared to using node directly.